libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -1973,9 +1973,9 @@ testUTXOWsubset,
     Assertion
 
 -- | Use an equality test on the expected and computed [PredicateFailure]
-testUTXOW wit@(UTXOW (Alonzo _)) = testUTXOWwith wit genericCont
-testUTXOW wit@(UTXOW (Babbage _)) = testUTXOWwith wit genericCont
-testUTXOW (UTXOW other) = error ("Cannot use testUTXOW in era " ++ show other)
+testUTXOW wit@(UTXOW (Alonzo _)) utxo = testUTXOWwith wit genericCont utxo
```

Comment:
I wounder if you inherited this from one of the commits in this PR that we want to remove...